### DISCUSSION TITLE
access open EXternal app

### DISCUSSION THREAD

**Discussion by iqmtgar** [cefsharp/CefSharp#4205] [General]
Hello,

Failed to run app using cefsharp browser
How do I give permission to run a program through  cefsharp browser

![image](https://user-images.githubusercontent.com/33340985/184509868-5ddaec66-41dd-4023-9474-f9bf13d3ade7.png)

Thank you for these great services

**Comment by amaitland** [comment]
In future please use discussions to ask questions. https://github.com/cefsharp/CefSharp/discussions

See https://github.com/cefsharp/CefSharp/wiki/General-Usage#protocol-execution
